cmd/pubsub_receive_adapter/main.go

```diff
@@ -0,0 +1,78 @@
+/*
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"log"
+	"os"
+
+	gcppubsub "github.com/GoogleCloudPlatform/cloud-run-events/pkg/adapter"
+
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+
+	"golang.org/x/net/context"
+)
+
+const (
+	// Environment variable containing project id
+	envProject = "GCPPUBSUB_PROJECT"
```

Comment:
Should all of these have the GCP prefix removed? Except possibly GCP_PROJECT.

Reply:
ignore these, they will all change when I integrate the sdk.